ARM-wvd-templates/HCI/HybridCompute/AddHciVirtualMachinesHybriComputeTemplate.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
nit:  the name of this file has a typo. on purpose?   HybriCompute.  its missing the "D" in hybrid